master/getting-started/docker/installation/vagrant-coreos/index.md

```diff
@@ -61,7 +61,7 @@ And finally check that Docker is running on both hosts by running
 
 ## 2. Try out Calico networking
 
-Now you have a basic two node CoreOS cluster setup, see the [Calico with Docker Networking Walkthrough]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/basic) for information on how to try Calico Networking.
+Now you have a basic two node CoreOS cluster setup, see the [Calico with Docker Networking Walkthrough]({{site.baseurl}}/{{page.version}}/getting-started/docker/installation/manual) for information on how to install Calico Networking.
```

Comment:
The VagrantCoreos guide just sets up a few hosts with calicoctl downloaded. You should just be able to run the standard install on them.